pkg/cluster/config/encoding/scheme.go

```diff
@@ -53,10 +54,16 @@ func AddToScheme(scheme *runtime.Scheme) {
 // If path == "" then the default config is returned
 func Load(path string) (*config.Cluster, error) {
 	var latestPublicConfig = &v1alpha3.Cluster{}
+	var err error
+	var contents []byte
```

Comment:
nit: we should not hoist these to a wider scope unnecessarily. these can be inside the `if path != "" {` block